health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignType.js

```
@@ -92,116 +157,120 @@ const CampaignSelection = ({ onSelect, formData, formState, ...props }) => {
+                </LabelFieldPair>
+              )}
+            </Card>
+          </div>
+          {showPopUp && (
+            <PopUp
+              className={"boundaries-pop-module"}
+              type={"default"}
+              heading={t("ES_CAMPAIGN_UPDATE_TYPE_MODAL_HEADER")}
+              children={[
+                <div>
+                  <CardText style={{ margin: 0 }}>{t("ES_CAMPAIGN_UPDATE_TYPE_MODAL_TEXT") + " "}</CardText>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)